README.md

```diff
@@ -3,9 +3,12 @@
 This project contains a fully integrated Hadoop+Search demo provisioned by Vagrant
 
 It automatically provisions three virtual machines running CentOS 6:
+
 * one node hosting a Couchbase server
 * one node hosting an Elasticsearch server
-* a fully functional, one node Hadoop cluster running Hortonworks HDP
+* one node hosting two different search applications
+
+There is also a fully functional one node Hadoop cluster running Hortonworks HDP for live streaming and processing of Twitter data. This machine is by default turned OFF as it takes awhile to provision, and automatically installing HDP is not highly reliable. Instructions for provisioning this machine into the data flow are listed below.
```

Comment:
Yes, in describing what is in this demo I believe this is the right spot to mention that there is a fourth machine, that is off by default. The instructions for actually using that machine are further down in the correct flow of execution for a user.